docs/networking/rocm.rst

```diff
@@ -0,0 +1,119 @@
+ROCm
+====
+
+How do I build Open MPI with ROCm-aware support?
+------------------------------------------------
+
+ROCm-aware support means that the MPI library can send and receive GPU
+buffers directly. As of today, ROCm support is available through
```

Comment:
Do you need to differentiate this text (and possibly the CUDA text?) that ROCm is specific to AMD hardware, and CUDA is specific to NVIDIA hardware?  (or whatever CUDA's restrictions are)